grakn-engine/src/main/java/ai/grakn/engine/controller/GraqlController.java

```diff
@@ -108,6 +109,35 @@ public GraqlController(EngineGraknGraphFactory factory, Service spark) {
         spark.exception(InvalidGraphException.class, (e, req, res) -> handleError(422, e, res));
     }
 
+    @POST
+    @Path("/execute")
+    @ApiOperation(value = "Execute an arbitrary Gralql queryEndpoints used to query the graph by ID or Graql match query and build HAL objects.")
+    @Produces({"application/json", "text/plain"})
```

Comment:
Yes, true, but I will remove this line, I think it's a leftover while I was troubleshooting why the method wasn't called at some point. The same annotation is specified at the class level and it's absent in all other methods. I doubt Spark is using it at all.